modules/tinymce/src/themes/silver/test/ts/browser/editor/header/PromotionTest.ts

```diff
@@ -0,0 +1,35 @@
+import { UiFinder } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { SugarElement } from '@ephox/sugar';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.header.PromotionTest', () => {
```

Comment:
Can we please make sure to add a `Assertions.assertStructure` check to these to ensure the DOM is correct for the promotion given my concern in the previous comment?